vscode-web/src/vs/editor/common/config/editorOptions.ts

```diff
@@ -3659,7 +3682,7 @@ export interface IGuidesOptions {
 	 * Enable highlighting of the active indent guide.
 	 * Defaults to true.
 	 */
-	highlightActiveIndentation?: boolean;
+	highlightActiveIndentation?: boolean | 'always';
```

Comment:
explicit defn. is required? or we can drop this?

Reply:
Thank you for you suggestion, the code here is sync from vscode, I think we can make it we can make it consistent with the upstream for future updates.